documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -705,6 +703,116 @@ See the {link-prefix}:{link-mongodb-connector}#mongodb-connector-properties[comp
 
 This configuration can be sent via POST to a running Kafka Connect service, which will then record the configuration and start up the one connector task that will connect to the MongoDB replica set or sharded cluster, assign tasks for each replica set, perform a snapshot if necessary, read the oplog, and record events to Kafka topics.
 
+[[mongodb-adding-connector-configuration]]
+=== Adding connector configuration
+
+ifdef::community[]
+To run a {prodname} MongoDB connector, create a connector configuration file and add it to your Kafka Connect cluster. 
+
+.Prerequisites
+
+* MongoDB is set up to run a {prodname} connector.
+
+* A {prodname} MongoDB connector is installed. 
+
+.Procedure
+
+. Create a configuration file for the MongoDB connector.
+
+. Use the link:{link-kafka-docs}/#connect_rest[Kafka Connect REST API] to add that connector configuration to your Kafka Connect cluster. 
+
+endif::community[]
+
+ifdef::product[]
+You can use a provided {prodname} container to deploy a {prodname} MongoDB connector. In this procedure, you build a custom Kafka Connect container image for {prodname}, configure the {prodname} connector as needed, and then add your connector configuration to your Kafka Connect environment. 
+
+.Prerequisites
+
+* You have Podman installed and sufficient rights to create and manage containers.
+* You installed the {prodname} MongoDB connector archive. 
+
+.Procedure
+
+. Extract the {prodname} MongoDB connector archive to create a directory structure for the connector plug-in, for example: 
++
+[subs=+macros]
+----
+pass:quotes[*tree ./my-plugins/*]
+./my-plugins/
+├── debezium-connector-mongodb
+│   ├── ...
+----
+
+. Create and publish a custom image for running your {prodname} connector:
+
+.. Create a new `Dockerfile` by using `{DockerKafkaConnect}` as the base image. In the following example, you would replace _my-plugins_ with the name of your plug-ins directory:
++
+[subs=+macros]
+----
+FROM registry.redhat.io/amq7/amq-streams-kafka-25:1.5.0
```

Comment:
Don't we have variable/attribute for the Strimzi (and its Kafka) version?

Reply:
I don't see a variable in antora.yml.  We do have one downstream:
`DockerKafkaConnect: registry.redhat.io/amq7/amq-streams-kafka-25:1.5.0`
I am adding that to antora.yml with the next commit. I hope the different attribute name case is okay.